recipes/sdl_ttf/all/conanfile.py

```diff
@@ -40,25 +42,39 @@ def configure(self):
         self.settings.rm_safe("compiler.cppstd")
         self.settings.rm_safe("compiler.libcxx")
 
+        if Version(self.version) < "2.20.0":
+            del self.options.with_harfbuzz
+
     def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
         self.requires("freetype/2.12.1")
-        self.requires("sdl/2.26.0")
+        self.requires("sdl/2.26.1")
+        if self.options.get_safe("with_harfbuzz"):
+            self.requires("harfbuzz/6.0.0")
 
     def validate(self):
         if is_msvc(self) and self.info.options.shared:
             raise ConanInvalidConfiguration("sdl_ttf shared is not supported with Visual Studio")
         if Version(self.version).major != Version(self.dependencies["sdl"].ref.version).major:
             raise ConanInvalidConfiguration("sdl & sdl_ttf must have the same major version")
 
+        if Version(self.version) >= "2.20.0" and self.options.shared != self.dependencies["sdl"].options.shared:
+            raise ConanInvalidConfiguration("sdl & sdl_ttf must be build with the same options(shared or static)")
+
     def source(self):
         get(self, **self.conan_data["sources"][self.version],
             destination=self.source_folder, strip_root=True)
 
     def generate(self):
         tc = CMakeToolchain(self)
+        if Version(self.version) >= "2.20.0":
+            tc.variables["CMAKE_POSITION_INDEPENDENT_CODE"] = False
```

Comment:
@toge it means that fPIC is always disabled. Is there an error which you can't fix or is it really needed? It may cause restriction when using this library as static and linking to other libraries with fPIC, probably the compiler will complain and ask to build sdl again with -fPIC.

Reply:
oops
It is wrong code.

I want to honor conamn options "fPIC".
I will fix it to follow yours.

Reply:
fixed.

Reply:
Arigato!